backend/backend.go

```diff
@@ -135,6 +136,18 @@ func Initialize(config *Config) (*Backend, error) {
 	logger.Debug("Done initializing store")
 	b.Store = stor
 
+	_, err = stor.GetClusterID(b.ctx)
+	if err != nil {
+		switch err := err.(type) {
+		case *store.ErrNotFound:
+			if storeErr := stor.CreateClusterID(b.ctx, uuid.New().String()); storeErr != nil {
```

Comment:
A quick note to mention that `uuid.New()` can _panic_ if it's not able to read enough bytes from the source of randomness it ultimately uses (OS dependent). It's probably fine since it would panic early (during the backend initialization), but I thought it may be worth mentioning...

Reply:
Definitely good to mention. I'm not sure that we can do anything to avoid the panic in this case, but during initialization is probably the best place to do so if there was an issue.